### PR TITLE
[refactor](operator) We dont need OperatorBuilder to instantiate Operator

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**

### CODE REVIEW

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Comment by zhiqiang-hhhh** [comment]
run buildall

File: be/src/pipeline/exec/aggregation_sink_operator.h [hunk 1]
```diff
@@ -39,7 +39,7 @@ class AggSinkOperatorBuilder final : public OperatorBuilder<vectorized::Aggregat
     bool is_sink() const override { return true; }
 };
 
-class AggSinkOperator final : public StreamingOperator<AggSinkOperatorBuilder> {
```
Comment (yiguolei) [inline be/src/pipeline/exec/aggregation_sink_operator.h:42]: If the operator does not depend on operator builder, then should remove all builders.

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by wangbo — APPROVED** [review]
LGTM

**Pull request merged**